Dictionary.java

```diff
@@ -0,0 +1,49 @@
+import java.util.LinkedList;
+
+public class Dictionary {
+	private LinkedList<String> words;
+	private LinkedList<String> prefixes;
+
+	public Dictionary(LinkedList<String> words) {
+		this.words = words;
+		this.prefixes = new LinkedList<String>();

```

Comment:
I see, thank you for your advice. I've changed the constructor, please let me know if it's alright now.